examples/terms-modal-banner-s3/webassets/app.html

```diff
@@ -0,0 +1,127 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <script
+      id="miro-sdk2"
+      src="https://miro.com/app/static/sdk/v2/miro.js"
+    ></script>
+    <link
+      rel="icon"
+      href="data:image/gif;base64,R0lGODlhAQABAAAAACwAAAAAAQABAAA="
+    />
+  </head>
+  <body>
+    <script id="variables" src="variables.js"></script>
+    <script>
+      window.userInfo = null;
+      async function callApi(url, method, body, token) {
+        const reqHeaders = {
+          accept: "application/json",
+          "cache-control": "no-cache",
+          pragma: "no-cache",
+          //'authorization': token //Uncomment this line if you are using an authorization token
```

Comment:
is this token usage documented somewhere?

Reply:
added docs, thank for the call out!